### DISCUSSION TITLE
File Naming error

### DISCUSSION THREAD

**Discussion by malanchito** [aclap-dev/video-downloadhelper#133] [Help and Q&A]
### Your question

On the previous version when i clicked on quick download (using the browser) the name of the file was the same as the one shown here
![Capture](https://github.com/aclap-dev/video-downloadhelper/assets/7159457/5725c0f8-48eb-4a38-8d16-05efe3329354)
but now the name is a different that is completely useless so i have to rename manually the files
![Capture](https://github.com/aclap-dev/video-downloadhelper/assets/7159457/21083c8c-efd5-4ccc-be73-789f2a9d4d6f)

### Addon & CoApp version and details

Version 8.2.0.23 VdhCoApp 2.0.10

**Comment by Astraeus000** [comment]
Having a similar naming related issue. In my case though it downloads anything as "getvid" and it's read as a generic file...renaming it with the .mp4 file extension immediately makes it what it's supposed to be, but it's weird all the same...
![Untitled](https://github.com/aclap-dev/video-downloadhelper/assets/160296343/f15a7767-2cf3-428f-8a5d-817bf78716aa)

**Comment by paulrouget** [comment]
Hi! We are aware of that issue, and it will be fixed in the upcoming version (this weekend).

Sorry for the bug :|

**Reply by Astraeus000** [reply]
Oh no worries, this kind of stuff happens. It's genuinely the first trouble I've had with the program in years and it's a slight inconvenience at most, but cheers for the reply all the same.
